src/prefect/artifacts.py

```diff
@@ -173,6 +177,24 @@ async def format(self) -> str:
         return json.dumps(self._sanitize(self.table))
 
 
+class ProgressArtifact(Artifact):
+    progress: float
+    type: Optional[str] = "progress"
```

Comment:
1. Is this actually optional?
2. Can it be any string or should it be literally `progress`?

Reply:
I will admit I don't know the answers to these questions, I am following the pattern for the other artifact classes. Example: 
https://github.com/PrefectHQ/prefect/pull/13469/files/cd7dabfa677767af72f31011aab1b532648589f0#diff-d31b9bd5edbc5bbc128826da300d200c20db14b9fdc4485ae78329d436b7ec85R155

Reply:
Ah fair enough then 👍